modules/secure_storage_connector/outputs.tf

```diff
@@ -1,5 +1,5 @@
-output "azure_storage_key" {
```

Comment:
not using storage access keys anymore, instead storage is giving RBAC to the managed identity